pyseir/load_data.py

```diff
@@ -0,0 +1,340 @@
+import os
+import logging
+import pandas as pd
+import numpy as np
+import urllib.request
+import requests
+import re
+import io
+import zipfile
+import json
+from pyseir import OUTPUT_DIR
+
+
+DATA_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', 'data')
+
+
+def load_zip_get_file(url, file, decoder='utf-8'):
+    """
+    Load a zipfile from a URL and extract a single file.  Note that this is
+    not ideal and may fail for large files since the files must fit in memory.
+
+    Parameters
+    ----------
+    url: str
+        URL to read from.
+    file: str
+        Filename to pull out of the zipfile.
+    decoder: str
+        Usually None for raw bytes or 'utf-8', or 'latin1'
+
+    Returns
+    -------
+    file_buffer: io.BytesIO or io.StringIO
+        The file buffer for the requested file if decoder is None else return
+        a decoded StringIO.
+    """
+    remotezip = urllib.request.urlopen(url)
+    zipinmemory = io.BytesIO(remotezip.read())
+    zf = zipfile.ZipFile(zipinmemory)
+    byte_string = zf.read(file)
+    if decoder:
+        string = byte_string.decode(decoder)
+        return io.StringIO(string)
+    else:
+        return io.BytesIO(byte_string)
+
+
+def cache_county_case_data():
+    """
+    Cache county covid case data from NYT in #PYSEIR_HOME/data.
+    """
+    logging.info('Downloading covid case data')
+    # NYT dataset
+    county_case_data = pd.read_csv('https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties.csv', dtype='str')
+    county_case_data['date'] = pd.to_datetime(county_case_data['date'])
+    county_case_data[['cases', 'deaths']] = county_case_data[['cases', 'deaths']].astype(int)
+    county_case_data = county_case_data[county_case_data['fips'].notnull()]
+    county_case_data.to_pickle(os.path.join(DATA_DIR, 'covid_case_timeseries.pkl'))
+
+#### Deprecated
```

Comment:
Left in for right now so I can recall how I calculated the county metadata and get that computed on your data feeds.  Will be gone before merging.